lib/ios/BackpackReactNative/Classes/FlareBridge/RCTBPKFlareManager.m

```diff
@@ -28,4 +29,6 @@ - (UIView *)view {
     return [[RCTBPKFlare alloc] initWithFrame:CGRectZero];
 }
 
+RCT_REMAP_VIEW_PROPERTY(pointerDirection, flarePosition, BPKFlarePosition)
```

Comment:
I have a feeling we generally prefix native properties that are mapped to RN with `rct_`. Maybe that's not always the case though
```suggestion
RCT_REMAP_VIEW_PROPERTY(pointerDirection, rct_flarePosition, BPKFlarePosition)
```

Reply:
I copied it from Dialog which isn't doing that.

My understanding of the API is:

```
RCT_REMAP_VIEW_PROPERTY(
  name_to_use_in_js,
  name_of_property_in_ios,
  type
)
```

If I'm correct we can't use anything other than `flarePosition` because that's the name of the property in iOS

Reply:
Ah yeah you're correct. We would use `rct_flarePosition` if we needed a middle value between js and the native code.